components/search/src/selector.rs

```diff
@@ -247,6 +247,23 @@ mod tests {
                         "base": "https://example.com/1",
                         "method": "GET",
                         "searchTermParamName": "q"
+                      },
+                      "suggestions": {
+                        "base": "https://example.com/suggestions",
+                        "method": "POST",
+                        "params": [{
+                          "name": "type",
+                          "value": "space",
+                        }],
+                        "searchTermParamName": "suggest"
+                      },
+                      "trending": {
+                        "base": "https://example.com/trending",
+                        "method": "GET",
+                        "params": [{
+                          "name": "area",
+                          "experimentConfig": "area-param",
```

Comment:
Wondering if we could we rename these to `trending-name` and `tending-experiment-value`?